src/yarp.c

```diff
@@ -9280,7 +9280,12 @@ parse_parameters(
 
                     yp_parser_local_add_token(parser, &parser->previous);
                     yp_forwarding_parameter_node_t *param = yp_forwarding_parameter_node_create(parser, &parser->previous);
-                    yp_parameters_node_keyword_rest_set(params, (yp_node_t *)param);
+                    if (params->keyword_rest == NULL) {
+                        yp_parameters_node_keyword_rest_set(params, (yp_node_t *)param);
```

Comment:
I love where you're going with this, but I actually want to favor the `...` here, and instead put the existing keyword rest into the `posts`. Could you move them around?

Reply:
Done in https://github.com/ruby/yarp/pull/1599/commits/5a912de16792577e926d54282b42dcfd9e8de47e